test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -202,6 +210,13 @@ void test_ucp_peer_failure::test_force_close()
     } else {
         EXPECT_FALSE(UCS_PTR_IS_ERR(close_req));
     }
+
+    mem_after = mallinfo();
+    /* Too low chance to predict memory consumption on wire up for all TLS */
+    if (GetParam().variant != FAIL_IMMEDIATELY) {
```

Comment:
seems like it checks if something was release, but not if everything was released?
does this test fail if w/o fix in UCP?

Reply:
yes, for udrc TL the consumption was constant